src/azul/indexer/index_service.py

```diff
@@ -449,8 +449,8 @@ def _aggregate_entity(self,
         contents = self._select_latest(contributions)
         aggregate_contents = {}
         for entity_type, entities in contents.items():
-            if entity_type == transformer.entity_type():
-                assert len(entities) == 1
+            if entity_type in transformer.inner_entity_types():
+                assert len(entities) <= 1
```

Comment:
This slightly weakens the assert. To get the same level of assurance, we would need to add something like
```
assert len((entity_type for entity_type in contents if entity_type in transformer.inner_entity_types())) == 1
```
Of course this would need to happen before the loop.